htmlToText.js

```diff
@@ -0,0 +1,14 @@
+
+/**
+ * Converts html text into plain text.
+ * @param {String} html HTML Content
+ */
+export const htmlToText = (html) => {
```

Comment:
This logic should be as simple as follows:

```js
htmlToText = (html) => {
  let d = document.createElement('div');
  d.innerHTML = html;
  return d.textContent;
}
```

In the logic you have written at present, you expect that on the next/subsequent call to this function, that element will be ready with the document and you don't need to create it again. But, that's not true. Because the newly created element is never attached to the document. Had you tested this scenario?

Reply:
This method can have 2nd param `trim` (Boolean), it's optional param. So, when missing it's value will be considered `false`. And based on that param return value should be trimmed or not.